MIT.md

```diff
@@ -1,8 +1,5 @@
 ## Copyright 2021, [Khaled alaa]
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
-
 Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
 
```

Comment:
- [ ] You did well adding the Licenee file, but it is displaying an error when clicked. please check the file path and make sure it is viewable on click 👍